com.locomalito.MalditaCastilla.yaml

```
@@ -0,0 +1,44 @@
+app-id: com.locomalito.MalditaCastilla
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
```

Update runtime 20.08
```suggestion
runtime-version: '20.08'
```

---

> Update runtime 20.08

Hi, but the binary is for i386 only, how do I make it work with an x86_64 runtime?